modules/SilkSpawners/src/main/java/de/dustplanet/silkspawners/events/SilkSpawnersSpawnerExplodeEvent.java

```diff
@@ -71,7 +77,8 @@
      * @param entityID new entity ID
      * @param dropChance the current dropChance
      */
-    public SilkSpawnersSpawnerExplodeEvent(@Nullable final Player player, final Block block, final String entityID, final int dropChance) {
+    public SilkSpawnersSpawnerExplodeEvent(Entity entity, @Nullable final Player player, final Block block, final String entityID, final int dropChance) {
```

Comment:
Thanks, your addition makes sense. To remain backwards compatible, can you add a constructor overload without the entity and add a `@Nullable` annotation? Otherwise, I can add this overload, too.

Reply:
Done